src/plugin/fieldViews/ProseMirrorFieldView.ts

```diff
@@ -197,7 +197,8 @@ export abstract class ProseMirrorFieldView extends FieldView<string> {
     const diffStart = node.content.findDiffStart(state.doc.content);
     const diffEnd = node.content.findDiffEnd(state.doc.content);
 
-    if (diffStart && diffEnd) {
+    // Check for null specifically, rather than falsiness, because a diff starting at pos 0 is a valid diff
+    if (diffStart !== null && diffEnd) {
```

Comment:
Good spot! Might be nice to also check diffEnd is not `null`? Who knows, maybe at some point negative positions will be valid, and `diffEnd` could be zero but still greater than `diffStart`.

Reply:
Done, `!== null` does seem like what we really want to check for there.